strings/strings.json

```diff
@@ -2682,6 +2682,18 @@
     "en": "Expand {0} facet",
     "fr": "Agrandir la facette {0}"
   },
+  "ShowLessFacetResults": {
+    "en": "Show less results for {0} facet",
+    "fr": "Montrer moins de résultats pour la facette {0}"
+  },
+  "ShowMoreFacetResults": {
+    "en": "Show more results for {0} facet",
+    "fr": "Montrer plus de résultats pour la facette {0}"
+  },
+  "SearchFacetResults": {
```

Comment:
Mmm... what are the "results" in this context? Are we not rather searching _for values in_ a facet?

Reply:
True! maybe value would be a better name indeed

Reply:
Then I suggest:

- Search for values in {0} facet
- Rechercher des valeurs dans la facette {0}